language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -82,79 +115,154 @@ mergePackageMap ps = foldM merge Map.empty ps
                     pure (Just (mbOldPkgName <|> mbPkgName, pkg))
         Map.alterF update pkgId pkgs
 
+-- Write packages for all the DALFs in all the DARs.
 main :: IO ()
 main = do
     opts@Options{..} <- execParser optionsParserInfo
+    let damlTypesVersion = fromMaybe "0.0.0" optDamlTypesVersion
```

Comment:
Rather than defaulting to `0.0.0`, I would call `getSdkVersion` here and remove the logic from `daml-helper` as mentioned above.

Reply:
Yep.